sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/EventProcessor{TPartition}.cs

```diff
@@ -452,6 +444,70 @@ protected EventProcessor()
                                                                            TimeSpan ownershipExpiration) =>
             new PartitionLoadBalancer(storageManager, identifier, consumerGroup, fullyQualifiedNamespace, eventHubName, ownershipExpiration);
 
+        /// <summary>
```

Comment:
This is just an ordering change; the method body has not been altered but it was not following the organization used by other source files in the project.